doc/devel/coding_style_guideline.rst

```diff
@@ -52,56 +54,60 @@ example, interleaved with multi-line comments that contain explanations of the
 <http://www.sphinx-doc.org/en/stable/rest.html>`_ for comments. Sphinx parses
-the files to produce the contents that are later rendered in the dipy_ website.
+the files to produce the contents that are later rendered in the dipy_
+website.
 
 The Python examples are compiled, output images produced, and corresponding
 ``.rst`` files produced so that the comments can be appropriately displayed
 in a web page enriched with images.
 
-Particularly, in order to ease the contribution of examples and ``.rst`` files,
-and with the consistency criterion in mind, beyond the numpy docstring standard
-aspects, contributors are encouraged to observe the following guidelines:
-* The acronym for the Diffusion Imaging in Python toolkit should be written as
-**DIPY**.
-* The classes, objects, and any other construct referenced from the code should
-be written with inverted commas, such as in *In DIPY, we use an object called
-``GradientTable`` which holds all the acquisition specific parameters, e.g.
-b-values, b-vectors, timings and others.*
-* Cite the relevant papers. Use the *[NameYear]* convention for
-cross-referencing them, such as in [Garyfaillidis2014]_, and put them under the
-:ref:`references` section.
-* Cross-reference related examples and files. Use the ``.. _specific_filename:``
-convention to label a file at the top of it. Thus, other pages will be able to
-reference the file using the standard Sphinx syntax
-``:ref:`specific_filename```.
-* Use an all-caps scheme for acronyms, and capitalize the first letters of the
-long names, such as in *Constrained Spherical Deconvolution (CSD)*, except in
-those cases where the most common convention has been to use lowercase, such
-as in *superior longitudinal fasciculus (SLF)*.
-* As customary in Python, use lowercase and separate words with underscores for
-filenames, labels for references, etc.
-* When including figures, use the regular font for captions (i.e. do not use
-bold faces), unless otherwise required for a specific text part (e.g. a DIPY
-object, etc.).
-* When referring to relative paths, use the backquote inline markup convention,
-such as in ``doc/devel``. Do not add the greater-than/less-than signs to
-enclose the path.
+Particularly, in order to ease the contribution of examples and ``.rst``
+files, and with the consistency criterion in mind, beyond the numpy
+docstring standard aspects, contributors are encouraged to observe the
+following guidelines:
+
+  * The acronym for the Diffusion Imaging in Python toolkit should be written
+    as **DIPY**.
+  * The classes, objects, and any other construct referenced from the code
+    should be written with inverted commas, such as in *In DIPY, we use an
+    object called ``GradientTable`` which holds all the acquisition specific
+    parameters, e.g. b-values, b-vectors, timings and others.*
+  * Cite the relevant papers. Use the *[NameYear]* convention for
+    cross-referencing them, such as in [Garyfaillidis2014]_, and put them
```

Comment:
`Garyfallidis`instead of `Garyfaillidis`

Reply:
Thanks ! I inspected this visually yesterday without being able to tell why it was not being linked 😬